src/scene/renderer/shadow-renderer-directional.js

```diff
@@ -160,14 +162,29 @@ class ShadowRendererDirectional {
             this.renderer.updateCameraFrustum(shadowCam);
             this.shadowRenderer.cullShadowCasters(comp, light, lightRenderData.visibleCasters, shadowCam, casters);
 
-            // find out AABB of visible shadow casters
             const visibleCasters = lightRenderData.visibleCasters;
-            const numVisibleCasters = visibleCasters.length;
+            const origNumVisibleCasters = visibleCasters.length;
+
+            let numVisibleCasters = 0;
+
+            // exclude all mesh instances that are hidden for this cascade.
+            // find out AABB of visible shadow casters
+
+            if (origNumVisibleCasters > 0) {
+                visibleSceneAabb.copy(emptySceneAabb);
```

Comment:
this is not a good solution, as that will cause all aabb's to include point 0,0,0.
Lets go back to what it was.